test/systemtests/network_test.go

```diff
@@ -89,6 +121,10 @@ func (s *systemtestSuite) TestNetworkAddDeleteVLAN(c *C) {
 	s.testNetworkAddDelete(c, "vlan")
 }
 
+func (s *systemtestSuite) TestNetworkAddDeleteWithDns(c *C) {
```

Comment:
This particular issue was just to make sure the skydns container shows up in netctl network inspect. So the encap on the network side doesnt really matter. If we want to create a separate dedicated set of tests for DNS operation itself, that should probably a new issue and there we can test both VLAN and VXLAN mode. So maybe testing separately for VLAN and VXLAN can be skipped here since it is independent of this particular issue.